src/screens/DiscoveryScreenV2/hooks/useDiscovery.js

```diff
@@ -21,17 +26,43 @@ const useDiscovery = () => {
       });
 
       await getFollowingTopic().then((response) => {
-        console.log({response}, 'likak');
         following.setFollowingTopics(response.data, followingDispatch);
       });
       setRefreshing(false);
     } catch (e) {
       setRefreshing(false);
     }
   };
+
+  const onRefreshDiscoveryUser = async () => {
+    try {
+      const discoveryInitialUserResponse = await DiscoveryRepo.fetchInitialDiscoveryUsers();
+      DiscoveryAction.setDiscoveryInitialUsers(
+        discoveryInitialUserResponse.suggestedUsers,
+        discoveryDispatch
+      );
+    } catch (e) {
+      setRefreshing(false);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, there's a lack of error handling in this try-catch block. Consider logging the error or handling it appropriately.
```diff
-     } catch (e) {
-       setRefreshing(false);
+     } catch (error) {
+       console.error(error);
+       setRefreshing(false);
      }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->